File: MedLabO.Models/Requests/Termin/TerminTestRezultatRequest.cs

namespace MedLabO.Models.Requests.Termin
{
    public class TerminTestRezultatRequest
    {
        public Guid TerminID { get; set; }
        public List<string> TestIDs { get; set; }
        public List<Rezultat> Rezultati { get; set; }
    }
}

File: MedLabO UI/medlabo_desktop/lib/models/termin/termin_test_rezultat_request.dart
import 'package:json_annotation/json_annotation.dart';
import 'package:medlabo_desktop/models/rezultat/rezultat.dart';

part 'termin_test_rezultat_request.g.dart';

@JsonSerializable()
class TerminTestRezultat {
  String? terminID;
  List<String>? testIDs;
  List<Rezultat>? rezultati;

  TerminTestRezultat({
    this.terminID,
    this.testIDs,
    this.rezultati,
  });

  factory TerminTestRezultat.fromJson(Map<String, dynamic> json) =>
      _$TerminTestRezultatFromJson(json);

  Map<String, dynamic> toJson() => _$TerminTestRezultatToJson(this);
}
